opentreemap/api/tests.py

```diff
@@ -373,7 +373,8 @@ def test_locations_plots_endpoint_max_plots_param_must_be_a_number(self):
         response = get_signed(
             self.client,
             "%s/instance/%s/locations/0,0/plots?max_plots=foo" % (
-                API_PFX, self.instance.url_name))
+                API_PFX, self.instance.url_name)
```

Comment:
It looks like this change has been reduced to moving a parenthesis. Consider removing it.

Reply:
Good catch. I've dropped the associated commit where this change stood alone.